docs/faq.md

```diff
@@ -115,9 +115,10 @@ We're looking to support:
 Referenced throughout this repo, the term "UWP" references a
 [Universal Windows Platform App](https://docs.microsoft.com/en-us/windows/uwp/get-started/) -
 an app running inside an app container, at a lower privilege level, with
-package identity, delivered via an MSIX. This app's lifecycle from activation
-through termination is controlled by the platform for best power and resource
-usage. Its access to the user's resources are gated by a Request model so the
+package identity, delivered via an MSIX. It is the only app type supported on Windows devices 
+such as Xbox, HoloLens, Surface Hub (Windows 10 Team) and devices running Windows 10 IoT Core.
```

Comment:
While this sentence is correct, please move it from here to a separate paragraph so the flow of "what defines the app type" isn't mixed with "where might the app type be useful."

Reply:
Nit: Surface Hub can be configured to run alternate Windows 10 SKUs that *do* allow for other app types.

I think the fact these restrictions are arbitrary and could be changed at any time, is a good reason it's not appropriate for inclusion in the document. It's more documentation debt.